zipline/utils/cache.py

```diff
@@ -290,11 +290,7 @@ def path(self):
     def _commit(self):
         """Sync the temporary file to the final path.
         """
-        self._tmpfile.close()
-        move(self._name, self._final_path)
-
-    def __getattr__(self, attr):
```

Comment:
It was unused, so I removed it.  This class itself was actually unused and had acquired some bitrot.